docs-gen/includes/dependent/penguin_training/pipeline.py

```diff
@@ -0,0 +1,46 @@
+import os
+from typing import List
+from tfx.components import CsvExampleGen, Trainer, ImportExampleGen
+from tfx.proto import trainer_pb2, example_gen_pb2
+from tfx.dsl.components.base.base_node import BaseNode
+from tfx.v1.dsl.experimental import RuntimeParameter
+from typing import Text
+
+def create_components() -> List[BaseNode]:
+    """Creates a penguin pipeline with TFX."""
+
+    # Defines a pipeline runtime parameter
+    examples_location_param = RuntimeParameter(
+        name = "examples_location",
+        ptype = Text
+    )
+
+    # imports data into the pipeline
```

Comment:
How is this different from "Brings data into the pipeline"? Also the other comments start with uppercase.